pkg/workload/core.go

```diff
@@ -136,13 +135,17 @@ func (c *CoreWorkload) Initialize() error {
 
 // Plan will create *Targets for each TargetName
 func (c *CoreWorkload) Plan(pt JobType, targets []string) error {
-	var isInterleaved bool
+	var needOperationMultiplication bool
 	apexTables := make([]schema.Table, 0)
 
 	// search func for looking if targets contains the given string
-	contains := func(s []string, searchterm string) bool {
-		i := sort.SearchStrings(s, searchterm)
```

Comment:
Using `sort.SearchStrings` is not appropriate since it assumes that input strings are already sorted. But this time it's not already guaranteed.

https://pkg.go.dev/sort#SearchStrings